pages/desktop/regions/filter.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, testsetup, lookup):
 
             # expand the thing here to represent the proper user action
             self._root_element = self.selenium.find_element(self._base_locator[0],
-                                    "%s/ul/li/a[normalize-space(text())='%s']" % (self._base_locator[1], lookup))
+                                                            "%s/ul/li/a[normalize-space(text())='%s']" % (self._base_locator[1], lookup))
```

Comment:
I'd suggest breaking on the first argument to make this line not so long. I prefer using the new string formatting over `%` formatting also but that isn't a huge deal.

```py
self._root_element = self.selenium.find_element(
    self._base_locator[0],
    "{base}/ul/li/a[normalize-space(text())='{lookup}']".format(
        base=self._base_locator[1], lookup=lookup))